R/explore_w_histograms.R

```diff
@@ -1,12 +1,97 @@
-#' take a csv file, a string of numerical column names
-#' and returns a png file of histogram(s)
+#' Helper function used to read the file and return a pandas dataframe.
+#' Checks if file type is a .csv or excel. If not, returns a ValueError.
```

Comment:
Does this return a ValueError? I know it does in Python, but I'm not sure what it returns in R.

Reply:
Good catch! I will change it to returns a printed message!